spec/features/books/new_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+RSpec.describe "as a user" do
+  describe "when I visit the new book page" do
+    it "I see a form to create a new book" do
+      visit new_book_path
+
+      within("#new-book-form") do
+        expect(page).to have_field("Title:")
+        expect(page).to have_field("Number of Pages:")
+        expect(page).to have_field("Year Published:")
+        expect(page).to have_field("Author(s):")
+        expect(page).to have_field("Image URL:")
+
+        expect(page).to have_button("Add Book")
+      end
+    end
+
+    it "I see a link to navigate back to book index" do
+      visit new_book_path
+
+      expect(page).to have_content("<< Return to All Books")
```

Comment:
Need to add this to the New Review page as well. Unless we are thinking of adding this option contextually to our Nav whenever that happens?